logsink/logsink.hpp

```diff
@@ -49,6 +49,16 @@ class FileSink : public google::LogSink
       const char* message,
       size_t message_len);
 
+  virtual void send(
```

Comment:
Could you leave a comment above the old deprecated method declaration explaining the historical reasons for its existence, with mention of the relevant JIRA tickets? (DCOS-24365 and MESOS-9687)

Reply:
Actually, there are two reasons - added them in comments.